components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -197,6 +199,15 @@ void MAMMicrophysics::set_grids(
   // - extfrc: 3D instantaneous forcing rate [kg/m³/s]
   add_field<Computed>("mam4_external_forcing", scalar3d_extcnt, kg / m3 / s, grid_name);
 
+  // Register computed fields for tendencies due to gas phase chemistry
+  // - dvmr/dt: Tendencies for mixing ratios  [kg/kg/s]
+  diagnostic_gs_dvmrdt_ = m_params.get<bool>("output_mixing_ratio_tendency_due_to_gas_phase_chemistry", false);
+  if (diagnostic_gs_dvmrdt_)
+    add_field<Computed>("mixing_ratio_tendency_due_to_gas_phase_chemistry", scalar2d_gas_pcnst, kg / kg / s, grid_name);
+  diagnostic_aq_dvmrdt_ = m_params.get<bool>("output_mixing_ratio_tendency_due_to_aqueous_chemistry", false);
+  if (diagnostic_aq_dvmrdt_)
+    add_field<Computed>("mixing_ratio_tendency_due_to_aqueous_chemistry", scalar2d_gas_pcnst, kg / kg / s, grid_name);
```

Comment:
If the array also includes the tendencies of gases, please make sure the corresponding unit is also `kg/kg/s`. Otherwise, we follow the `constituent_flux` example.

Reply:
This array has gases and aerosols. We are following exactly how it is computed in the Fortran code, including the unit conversion.